paper-tooltip.html

```diff
@@ -140,16 +147,15 @@
        * of the tooltip.
        */
       get target () {
-        var ownerRoot = Polymer.dom(this).getOwnerRoot();
         var parentNode = Polymer.dom(this).parentNode;
-        var target;
+        // If the parentNode is a document fragment, then we need to use the host.
+        var ownerRoot = Polymer.dom(this).getOwnerRoot();
 
+        var target;
         if (this.for) {
-          target = parentNode.querySelector('#' + this.for);
-        } else if (parentNode.nodeType == 11) { // DOCUMENT_FRAGMENT_NODE
-          target = ownerRoot.host;
+          target = Polymer.dom(ownerRoot).querySelector('#' + this.for);
         } else {
-          target = parentNode;
+          target = parentNode.nodeType == 11 ? ownerRoot.host : parentNode;
```

Comment:
This is a little nit picky, but I think it's more sanitary to test against `Node.DOCUMENT_FRAGMENT_NODE` instead of the vague number `11`.